README.md

```diff
@@ -293,11 +293,11 @@ Default:        OS default
 
 ```
 Type:           bool / string
-Valid Values:   true, false, skip-verify, <name>
+Valid Values:   true, false, skip-verify, <name>, <empty>
```

Comment:
While I agree that we need a fix for `tls=true`, `<empty>` should not be a valid (/recommended) value. It doesn't make sense semantically and is the same as `tls=true`, I assume?